packages/modelviewer.dev/examples/postprocessing/index.html

```diff
@@ -0,0 +1,222 @@
+    <outline-effect blur-strength="2" edge-strength="4" color="teal"></outline-effect>
+    <!-- <pixelate-effect granularity="10"></pixelate-effect> -->
+    <!-- <glitch-effect strength=".5" mode="sporadic"></glitch-effect> -->
+    <smaa-effect quality="ultra"></smaa-effect>
+    <ssao-effect></ssao-effect>
+    <!-- <bloom-effect id="bloom"></bloom-effect>
+    <color-grade-effect id="color-grade" contrast="0.1" hue="0" saturation="0.75"></color-grade-effect> -->
+  </effect-composer>
+  <div class="controls">
+    <label for="intensity">Intensity</label>
+    <input id="intensity" type="range" min="0" max="5" step="0.1" value="1">
+    <label for="threshold">Threshold</label>
+    <input id="threshold" type="range" min="0" max="1" step="0.01" value="0.85">
+    <label for="smoothing">Smoothing</label>
+    <input id="smoothing" type="range" min="0" max="1" step="0.01" value="0.02">
+  </div>
+</model-viewer>
+<script>
+const mv = document.querySelector("model-viewer#bloomViewer");
+const bloom = mv.querySelector('#bloom');
+const color = mv.querySelector('#color-grade');
+
+/*if (color) {
+  const a = setInterval(() => {
+    if (!color.hue) color.hue = 0;
+    color.hue += .01;
+  }, 10);
+}*/
+
+function updateProperty(event, propertyName) {
+  bloom[propertyName] = event.target.value;
+}
+
+mv.querySelector('#intensity').addEventListener('input', (e) => updateProperty(e, 'intensity'));
+mv.querySelector('#threshold').addEventListener('input', (e) => updateProperty(e, 'threshold'));
+mv.querySelector('#smoothing').addEventListener('input', (e) => updateProperty(e, 'smoothing'));
+</script>
+            </template>
+          </example-snippet>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="footer">
+      <ul>
+        <li class="attribution">
+          <a href="https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Lantern">Lantern</a>,
+          licensed under <a href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons Zero</a>.
+        </li>
+      </ul>
+      <div style="margin-top:24px;" class="copyright">©Copyright 2018-2020 Google Inc. Licensed under the Apache License 2.0.</div>
+      <div id='footer-links'></div>
+    </div>
+  </div>
+</div>
+
+  <script type="module" src="../../examples/built/docs-and-examples.js">
+  </script>
+  <script type="module">
+    (() => { init('examples-postprocessing'); })();
+    (() => { initFooterLinks();})();
+  </script>
+
+  <!-- Documentation-specific dependencies: -->
+  <script type="module"
+      src="../built/dependencies.js">
+  </script>
+
+  <!-- Loads <model-viewer> on modern browsers: -->
+  <script type="module"
+      src="../../node_modules/@google/model-viewer/dist/model-viewer-no-three.min.js">
+  </script>
+
+  
+  <!-- Loads <model-viewer-effects> on modern browsers: -->
+  <script type="module"
+      src="../../node_modules/@beilinson/model-viewer-effects/dist/model-viewer-effects.js">
```

Comment:
>The postprocessing examples are already using the import-map, do you have any input on how to call the no-three package bundle?
>Where do you think the documentation and examples for this library should sit btw? If it's on the base modelviewer website that'd be easiest so I can use all the same templates you're using.

Honestly, this example you've built is actually doing exactly what I wanted to see, and I think this is a great place for it. We'll just need to link it in as another examples page. I like this because it's clear there are three scripts coming in instead of one: `three.module.js` (in the import map), `model-viewer-no-three.min.js`, and `model-viewer-effects.js`. What are the sizes of those (gzipped & minified)?